src/pages/index.tsx

```diff
@@ -1,18 +1,30 @@
-import Head from 'next/head';
-import Image from 'next/image';
-
-import backgroundSVG from '@/images/background-hero.svg';
-
 import { CallToAction } from '@/components/CallToAction';
 import { Faqs } from '@/components/Faqs';
 import { Footer } from '@/components/Footer';
+import Head from 'next/head';
 import { Header } from '@/components/Header';
 import { Hero } from '@/components/Hero';
+import Image from 'next/image';
+import Lenis from 'lenis'
 import { PrimaryFeatures } from '@/components/PrimaryFeatures';
 import { SecondaryFeatures } from '@/components/SecondaryFeatures';
 import { Testimonials } from '@/components/Testimonials';
+import backgroundSVG from '@/images/background-hero.svg';
+import { useEffect } from 'react';
 
 export default function Home() {
+
+  useEffect(() => {
```

Comment:
This only applied this for the homepage right? Is it possible to move it somewhere else so that it applies to all other pages as well automatically?